src/99-appendices/08-coordinate-systems.md

```diff
@@ -151,6 +151,20 @@ Restricted keywords for the `<CoordSysType>CoordinateSystem` field in the
     [ACPC site](https://www.fieldtriptoolbox.org/faq/acpc/) on the FieldTrip
     toolbox wiki.
 
+-   `fsnative`: The subject-specific FreeSurfer T1w space. This corresponds to
+    the original scanner image after it is re-sampled and re-scaled
+    to an isotropic 1 mm 256x256x256 volume in RAS orientation. The origin
+    is wherever the original origin of the ``orig.mgz`` file is. One MUST
+    align the origin to the ``ACPC`` before using this coordinate system.
+
+-   `fsnativetkr`: The subject-specific FreeSurfer T1w space. The origin of
+    the coordinate system is at the center of isotropic 1 mm 256x256x256
+    volume (that is the voxel center is at slices (128, 128, 128)) in RAS
```

Comment:
are these indices 0-offset or 1-offset? 

Should the middle of a 256x256x256 volume not be represented as [127.5 127.5 127.5] (when 0-offset) or  [128.5 128.5 128.5] (where 1-offset)?

The middle of a 2x2x2 volume would be at voxel [0.5 0.5 0.5] if you start counting voxels at [0 0 0], or at  [1.5 1.5 1.5] when you start counting at [1 1 1].

Reply:
I'm pretty sure it is with 0-offset.

That's what I thought initially... but I think there is some offset, so for example, running the following

```
        mri_info --vox2ras-tkr <img>

    This will generally be the 4x4 matrix for FreeSurfer output.::

            [
                [-1.0, 0.0, 0.0, 128.0],
                [0.0, 0.0, 1.0, -128.0],
                [0.0, -1.0, 0.0, 128.0],
                [0.0, 0.0, 0.0, 1.0],
            ]
```

This essentially centers the coordinates at the "center" of the volume "almost".

Reference: https://www.mail-archive.com/freesurfer@nmr.mgh.harvard.edu/msg69541.html